conf/postgres-operator/affinity.json

```diff
@@ -1,7 +1,6 @@
-        "affinity": {
             "nodeAffinity": {
             "preferredDuringSchedulingIgnoredDuringExecution": [{
-                "weight": 1,
+                "weight": 10,
```

Comment:
This appears to be set to `10` whereas in the Ansible role it's still at `1`

Reply:
Updated the `weight` to `10` in the Ansible installer to align with the setting here.  This is to give the node affinity precedence when node preference is being determined.